web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -386,6 +390,77 @@ describe("profile - shared", () => {
     });
   });
 
+  describe("Special Note Alert for Businesses Formed outside the Navigator", () => {
```

Comment:
Can we remove `Special` from the test description or add it to the others below, so it doesn't seem like it's two different notes (e.g. a special one and a not-so-special one)?